packages/yourloops/app/app.tsx

```diff
@@ -69,14 +72,17 @@ class Yourloops {
 
   private router(): JSX.Element {
     return (
-      <Router>
-        <Switch>
-          <Route exact path="/" component={LoginPage} />
-          <Route path="/login" component={LoginPage} />
-          <Route path="/hcp" component={HcpPage} />
-          <Route path= "/patient" component={PatientPage} />
-        </Switch>
-      </Router>
+      <AuthProvider>
+        <Router>
+          <Switch>
+            <Route exact path="/" component={LoginPage} />
+            <Route path="/login" component={LoginPage} />
```

Comment:
I would not have 2 routes for the same page, for me the /login route seems useless.
```suggestion
```

Reply:
yes you are right